conanfile.py

```diff
@@ -18,21 +21,37 @@ class TFLiteConan(ConanFile):
     exports = ["01-tar-instead-of-zip.patch", "02-ignore-nnapi.patch"]
     settings = "os", "arch", "compiler", "build_type"
 
-    build_subfolder = "tensorflow/lite/tools/make"
+    def system_requirements(self):
+        installer = tools.SystemPackageTool()
+        installer.install("make")
```

Comment:
is this required for running `autotools.make` below?

Reply:
That was the idea, but on OSX it was giving more issues than benefits so I just removed it. (`make` is a pretty basic thing to have anyway)